torchvision/models/detection/rpn.py

```diff
@@ -145,8 +141,8 @@ def grid_anchors(self, grid_sizes, strides):
         return anchors
 
     def cached_grid_anchors(self, grid_sizes, strides):
-        # type: (List[List[int]], List[List[int]])
-        key = str(grid_sizes + strides)
+        # type: (List[List[int]], List[List[Tensor]])
+        key = str(grid_sizes) + str(strides)
```

Comment:
@lara-hdr Maybe the comment was deleted after update.
So the grid_sizes is a list of ints, and strides is a list of tensors. So we cannot concat and then cast them to string. But casting to string and concatenating the strings has the same result.